### PR TITLE
Libs 8.0.0 release

### PR DESCRIPTION
**Pull request by iruzevic**
### Updated
- `Eightshift-forms-utils` to the latest version `2.0.0`.
- `@infinum/eightshift-libs` to the latest version `8.0.0`.

### Removed
- `MAIN_PLUGIN_MANIFEST_ITEM_HOOK_NAME` constant.
- `getDataManifest`, `getDataManifestRaw`, `getProjectVersion`, `getCountrySelectList` functions.
- `src/Manifest/UtilsManifest.php` class.

### Changed
- `getDataManifestPath` helper now supports only `$path` param.
- `camelToSnakeCase`, `kebabToSnakeCase`, `recursiveFind`, `getCurrentUrl`, `cleanPageUrl` are now used from the `@infinum/eightshift-libs` package.

# TODO
- [x] Changelog
- [ ] Release
- [ ] Composer remove temp items

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
 
 This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a CHANGELOG](https://keepachangelog.com/).
 
+## [2.0.0]
+
+### Updated
+- `Eightshift-forms-utils` to the latest version `2.0.0`.
```
Comment (goranalkovic-infinum) [inline CHANGELOG.md]:
```suggestion
- `eightshift-forms-utils` to version `2.0.0`.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
...
+- `@infinum/eightshift-libs` to the latest version `8.0.0`.
```
Comment (goranalkovic-infinum) [inline CHANGELOG.md]:
```suggestion
- `@infinum/eightshift-libs` to version `8.0.0`.
```

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]

**Review by goranalkovic-infinum — DISMISSED** [review]

File: composer.json [hunk 3]
```diff
@@ -32,9 +32,15 @@
...
+	"repositories": [
+		{
+			"type": "vcs",
+			"url": "https://github.com/infinum/eightshift-libs.git"
+		}
+	],
 	"require": {
 		"php": "^7.4 || >=8.0",
-		"infinum/eightshift-libs": "^7.1.2"
+		"infinum/eightshift-libs": "dev-feature/optimizations"
```
Comment (dingo-d) [inline composer.json]: I recon this will be updated to v 8, right?

**Review by dingo-d — DISMISSED** [review]
Just left one question 👍🏼

File: composer.json (hunk 3 above)
Comment (iruzevic) [reply composer.json]: correct once everything is finished with reviews and testing

**Review by piqusy — DISMISSED** [review]

**Pull request merged**